pipelines/docker-build-oci-ta/patch.yaml

```diff
@@ -0,0 +1,122 @@
+    "pipelines.openshift.io/strategy": "docker"
+- op: remove
+  path: /spec/workspaces/0
+# Order of Tasks from the base docker-build Pipeline:
+# $ kustomize build pipelines/docker-build | yq .spec.tasks.[].name | nl -v 0
+#  0  init
+#  1  clone-repository
+#  2  prefetch-dependencies
+#  3  build-container
+#  4  build-source-image
+#  5  deprecated-base-image-check
+#  6  clair-scan
+#  7  ecosystem-cert-preflight-checks
+#  8  sast-snyk-check
+#  9  clamav-scan
+# 10  sbom-json-check
+# 11  apply-tags
+
+# clone-repository Task
+- op: replace
+  path: /spec/tasks/1/taskRef/name
+  value: git-clone-oci-ta
+- op: add
+  path: /spec/tasks/1/params/-
+  value:
+    name: ociStorage
+    value: $(params.output-image).git
+- op: add
+  path: /spec/tasks/1/params/-
+  value:
+    name: ociArtifactExpiresAfter
+    value: $(params.image-expires-after)
+- op: remove
+  path: /spec/tasks/1/workspaces/0
+
+# prefetch-dependencies Task
+- op: replace
+  path: /spec/tasks/2/taskRef/name
+  value: prefetch-dependencies-oci-ta
+- op: add
+  path: /spec/tasks/2/params/-
+  value:
+    name: SOURCE_ARTIFACT
+    value: $(tasks.clone-repository.results.SOURCE_ARTIFACT)
+- op: add
+  path: /spec/tasks/2/params/-
+  value:
+    name: ociStorage
+    value: $(params.output-image).prefetch
+- op: add
+  path: /spec/tasks/2/params/-
+  value:
+    name: ociArtifactExpiresAfter
+    value: $(params.image-expires-after)
+- op: remove
+  path: /spec/tasks/2/workspaces/0
+
+# build-container
+- op: replace
+  path: /spec/tasks/3/taskRef/name
+  value: buildah-oci-ta
+- op: add
+  path: /spec/tasks/3/params/-
+  value:
+    name: SOURCE_ARTIFACT
+    value: $(tasks.prefetch-dependencies.results.SOURCE_ARTIFACT)
+- op: add
+  path: /spec/tasks/3/params/-
+  value:
+    name: CACHI2_ARTIFACT
+    value: $(tasks.prefetch-dependencies.results.CACHI2_ARTIFACT)
+- op: remove
+  path: /spec/tasks/3/workspaces/0
+
+# build-source-image
+- op: replace
+  path: /spec/tasks/4/taskRef/name
+  value: source-build-oci-ta
+- op: add
+  path: /spec/tasks/4/params/-
+  value:
+    name: SOURCE_ARTIFACT
+    value: $(tasks.prefetch-dependencies.results.SOURCE_ARTIFACT)
+- op: add
+  path: /spec/tasks/4/params/-
+  value:
+    name: CACHI2_ARTIFACT
+    value: $(tasks.prefetch-dependencies.results.CACHI2_ARTIFACT)
+- op: remove
+  path: /spec/tasks/4/workspaces/0
+
+# sast-snyk-check
+- op: replace
+  path: /spec/tasks/8/taskRef/name
+  value: sast-snyk-check-oci-ta
+- op: add
+  # In the docker-build Pipeline, the snyk Task does not receive any parameters, so we cannot
+  # append to it.
+  path: /spec/tasks/8/params
+  value:
+    - name: SOURCE_ARTIFACT
+      value: $(tasks.prefetch-dependencies.results.SOURCE_ARTIFACT)
+- op: remove
+  path: /spec/tasks/8/workspaces/0
+
+# Order of finally Tasks from the base docker-build Pipeline:
+# $ kustomize build pipelines/docker-build | yq .spec.finally.[].name | nl -v 0
+#  0  show-sbom
+#  1  show-summary
+
+# show-summary
+- op: remove
```

Comment:
Maybe add a comment to explain why we want to nuke this one.

Reply:
Good call. Thanks.